### DISCUSSION TITLE
TinyBit Launcher discussions

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#64] [General]
[![Gitter](https://badges.gitter.im/TinyBit-launcher/community.svg)](https://gitter.im/TinyBit-launcher/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![Matrix](https://img.shields.io/matrix/TBLauncher:matrix.org?logo=Matrix)](https://matrix.to/#/#TBLauncher:matrix.org)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

.

**Comment by ildar** [comment]
Github Discussions is a new kid in the hood. This is the second first I
encounter it (today).
Yet many github projects willing to "discuss" usually go to Gitter and
happy there. Why didn't you?
BTW, Gitter recently improved the Matrix integration. The Matrix itself
provides almost limitless access to and from other chat platforms incl.
Telegram, IRC, whatnot. IMHO definitely the platform of choice.

**Reply by TBog** [reply]
I'm not familiar with Gitter.
This project is small and imho it's easy just to use one communication channel.
I'll try your idea and add the Gitter and Reddit links to the front page.

**Comment by ildar** [comment]
IMHO as TBLauncher is targeting mere users, it's wise to use a more popular
platform for the community, not the Github (which is mostly dev-oriented).
But that's obviously your choice only.
